backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -119,6 +126,16 @@
         return resources.createEndpoint(endpoint);
     }
 
+    @POST
+    @Path("/system/email_subscription")
```

Comment:
Not hidden. If you have a better name it would be good :-)

The purpose of this path is to group `Endpoint`s that are meant to me managed by us (EmailSubscription and NotificationDrawer maybe)